generators/frontend/templates/react-demo/components/Footer.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Link from 'next/link';
+import styled from 'styled-components';
+
+export default () => {
+  const Wrapper = styled.div`
+    color: #2f4f4f;
+    padding: 1.875rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    hr {
+      border-top: 0.125rem solid #f3f3f3;
+      border-left-color: transparent;
+      height: 0.375rem;
+      border-right-color: transparent;
+      border-bottom: 0.125rem solid #f3f3f3;
+      width: 74%;
+    }
+    div {
+      width: 100%;
+      .copyright {
+        font-size: 1.125rem;
```

Comment:
This font-size is repeated in many places. It's better to handle it with generic css classes